.env.example

```diff
@@ -10,3 +10,5 @@ CYBERSOURCE_TRANSACTION_KEY=
 CYBERSOURCE_SECURE_ACCEPTANCE_URL=
 CYBERSOURCE_PROFILE_ID=
 CYBERSOURCE_REFERENCE_PREFIX=
+MAILGUN_KEY=
+MAILGUN_RECIPIENT_OVERRIDE=
```

Comment:
@noisecapella these are the only two Mailgun settings values that really make sense to expect from someone running MM locally. let me know if you think something should be added